test/external/wannier.jl

```diff
@@ -0,0 +1,28 @@
+using Test
+using DFTK
+include("../testcases.jl")
+
+if !Sys.iswindows() && mpi_nprocs() == 1
```

Comment:
This gate was only needed because wannier90 is not available in Yggdrasil on windows. But I hope your code works there?

Reply:
I suppose one could save some code duplication by merging this with the wannier90 test.